src/app/core/view/dynamic-components/dynamic-component.decorator.ts

```diff
@@ -0,0 +1,15 @@
+import { DynamicRegistry } from "../../registry/DynamicRegistry";
+import { OnInitDynamicComponent } from "./on-init-dynamic-component.interface";
+
+export function DynamicComponent<
```

Comment:
short jsDoc summary please

Reply:
and we should update the documentation: https://aam-digital.github.io/ndb-core/documentation/additional-documentation/concepts/configuration.html#views